src/main/java/core/basesyntax/service/fruittransactionservice/FruitTransactionProcessorImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.fruittransactionservice;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class FruitTransactionProcessorImpl implements FruitTransactionProcessor {
+    private OperationStrategy operationStrategy;
+
+    public FruitTransactionProcessorImpl(OperationStrategy operationStrategy) {
+        this.operationStrategy = operationStrategy;
+    }
+
+    @Override
+    public void makeDailyFruitsUpdate(List<FruitTransaction> fruitTransactionList) {
+        fruitTransactionList.forEach(f -> operationStrategy.getOperation(f.getOperation())
```

Comment:
a more informative name is t. After all, we are processing transactions here
```suggestion
        fruitTransactionList.forEach(t -> operationStrategy.getOperation(t.getOperation())
```